### PR TITLE
Fix for STORE-1188, UI improvements to the store and publisher

### PR DESCRIPTION
**Pull request by tmkasun**
In this PR I have, 
- Move Show Dependency and Compare with (previously diff-view) buttons in to new button group
- Make version selection for comparison more intuitive
- Need to improve publisher side button colors with fix in issue https://wso2.org/jira/browse/STORE-1190

### CODE REVIEW

File: modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/asset.js [hunk 1]
```diff
@@ -106,6 +106,11 @@ asset.renderer = function(ctx) {
 
                 var type = page.assets.type;
                 page.assetVersions = gregAPI.getAssetVersions(ctx.session, ctx.assetType, page.assets.path, page.assets.name);
+                var single_version = false;
+                if(page.assetVersions.length === 1){
```
Comment (denuwanthi) [inline modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/asset.js]: formatting of If conditions needed

File: modules/es-extensions/store/extensions/app/greg-store-defaults/asset.js [hunk 2]
```diff
@@ -106,6 +106,11 @@ asset.renderer = function(ctx) {
 
                 var type = page.assets.type;
                 page.assetVersions = gregAPI.getAssetVersions(ctx.session, ctx.assetType, page.assets.path, page.assets.name);
+                var single_version = false;
+                if(page.assetVersions.length === 1){
```
Comment (daneshk) [inline modules/es-extensions/store/extensions/app/greg-store-defaults/asset.js]: formatting issue

File: modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/asset.js (hunk 1 above)
Comment (tmkasun) [inline modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/asset.js]: Thanks @denuwanthi , I have fixed it in [this commit](https://github.com/wso2/product-greg/pull/854/commits/6faf5fe5f797312d71e574912011f10a69d42c37)

File: modules/es-extensions/store/extensions/app/greg-store-defaults/asset.js (hunk 2 above)
Comment (tmkasun) [inline modules/es-extensions/store/extensions/app/greg-store-defaults/asset.js]: Thanks @daneshk , I have fixed it in [this commit](https://github.com/wso2/product-greg/pull/854/commits/6faf5fe5f797312d71e574912011f10a69d42c37)

File: modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/themes/default/js/diff-view.js [hunk 3]
```diff
@@ -1,9 +1,18 @@
-$("#diff-view-version").on('change',function(){
-    var diff_view_url = "/publisher/pages/diff?type=wsdl&path=" + $("#diff-view-version").val() + ',' +
-        $("#diff-view-version").find(':selected').data('base_path');
+$("#diff-view-version li").on('mouseover',function(event){
```
Comment (splinter) [inline modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/themes/default/js/diff-view.js]: Formatting issue , line 1

File: modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/js/diff-view.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+$("#diff-view-version li").on('mouseover',function(event){
```
Comment (splinter) [inline modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/js/diff-view.js]: Formatting issue, line 1

Comment (tmkasun) [inline modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/js/diff-view.js]: Thanks @splinter , I have fixed it in [this commit](https://github.com/wso2/product-greg/pull/854/commits/0f0795aa166ee07f2ea2ab7bd0b9bccfdde34e90)

File: modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/themes/default/js/diff-view.js (hunk 3 above)
Comment (tmkasun) [inline modules/es-extensions/publisher/extensions/app/greg-publisher-defaults/themes/default/js/diff-view.js]: Thanks @splinter , I have fixed it in [this commit](https://github.com/tmkasun/product-greg/commit/3a678f754aef8776cbf4f7854bf8588dd14e7730)

**Pull request merged**